rasa_core_sdk/endpoint.py

```diff
@@ -9,35 +9,41 @@
 from flask import Flask, jsonify, request
 from flask_cors import CORS, cross_origin
 from gevent.pywsgi import WSGIServer
-
 from rasa_core_sdk.executor import ActionExecutor
 
 DEFAULT_SERVER_PORT = 5055
 
 logger = logging.getLogger(__name__)
 
 
+def action_arg(action):
```

Comment:
I am split on this: should we do this or rather reject anything that contains `/`? The reason I am worried is, that the path components might not actually be proper python packages (e.g. missing `__init__.py`) - so the loading with the converted name will fail as well

Reply:
I think the missing __init__.py can happen I both cases, with the `/` or with the `.`. 
But if there are other reasons i can just reject anything with `/`, just let me know.

Reply:
yes it can definitely happen in both cases, but if the user provides it with `.` we can assume he knows it is supposed to be a python package name, if he provides it with `/` he might not know it should be a python package and the error message he will get will not make sense to him